docs/api_reference/src/quantities.tex

```diff
@@ -2812,6 +3101,723 @@
+template<std::@\stdconcept{derived_from}@<quantity> Q, auto R2, typename Rep2>
+  requires @\seebelownc@
+friend constexpr bool operator<=>(const Q& lhs, const quantity<R2, Rep2>& rhs);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+using ct = std::common_type_t<quantity, quantity<R2, Rep2>>;
+const ct ct_lhs(lhs);
+const ct ct_rhs(rhs);
+return ct_lhs.numerical_value_ref_in(ct::unit) @\atsign@ ct_rhs.numerical_value_ref_in(ct::unit);
+\end{codeblock}
+
+\pnum
+\remarks
+The expression in the \fakegrammarterm{requires-clause} is equivalent to:
+\begin{codeblock}
+requires requires {
+  typename std::common_type_t<quantity, quantity<R2, Rep2>>;
+} && @\placeholder{C}@<typename std::common_type_t<quantity, quantity<R2, Rep2>>::rep>
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{quantity}{operator==}
+\indexlibrarymember{quantity}{operator<=>}
+\begin{itemdecl}
+template<std::@\stdconcept{derived_from}@<quantity> Q, @\libconcept{RepresentationOf}@<quantity_character::scalar> Value>
+  requires @\seebelownc@
+friend constexpr bool operator==(const Q& lhs, const Value& rhs);
+template<std::@\stdconcept{derived_from}@<quantity> Q, @\libconcept{RepresentationOf}@<quantity_character::scalar> Value>
+  requires @\seebelownc@
+friend constexpr bool operator<=>(const Q& lhs, const Value& rhs);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{lhs.numerical_value_ref_in(unit) @ rhs}.
+
+\pnum
+\remarks
+The expression in the \fakegrammarterm{requires-clause} is equivalent to:
+\begin{codeblock}
+(Q::unit == ::mp_units::one) && @\placeholder{C}@<Rep, Value>
+\end{codeblock}
+\end{itemdescr}
+
+\rSec3[qty.non.mem.conv]{Non-member conversions}
+
+\begin{itemdecl}
+template<@\libconcept{Quantity}@ To, typename FwdFrom, @\libconcept{Quantity}@ From = std::remove_cvref_t<FwdFrom>>
+  requires @\seebelownc@
+constexpr To @\exposidnc{sudo-cast}@(FwdFrom&& q);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+TBD.
+\end{itemdescr}
+
+\rSec3[qty.common.type]{\tcode{std::common_type} specializations}
+
+\indexlibraryglobal{std::common_type}
+\begin{codeblock}
+template<mp_units::@\libconcept{Quantity}@ Q1, mp_units::@\libconcept{Quantity}@ Q2>
+  requires @\seebelownc@
+struct std::common_type<Q1, Q2> {
+  using type = mp_units::quantity<mp_units::get_common_reference(Q1::reference, Q2::reference),
+                                  std::common_type_t<typename Q1::rep, typename Q2::rep>>;
+};
+
+template<mp_units::@\libconcept{Quantity}@ Q,
+         mp_units::@\libconcept{RepresentationOf}@<mp_units::quantity_character::scalar> Value>
+  requires @\seebelownc@
+struct std::common_type<Q, Value> {
+  using type = mp_units::quantity<Q::reference, std::common_type_t<typename Q::rep, Value>>;
+};
+\end{codeblock}
+
+\pnum
+\remarks
+For the first specialization,
+the expression in the \fakegrammarterm{requires-clause} is equivalent to:
```

Comment:
I've seen this many times in the docs. Why can't we just provide requires-clause in the definition? Especially, when no exposition concepts are being used.

The same applies to `explicit` operator.

Reply:
It is possible, but is mostly done for aesthetic reasons.
The larger and more convoluted the expression,
the better it is off to the _Remarks_.